data/hai missions.txt

```diff
@@ -1061,3 +1061,251 @@ mission "Hai Honeymoon"
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		conversation
+			`Allhome's docking area shows much more traffic than usual. Many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You finally meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`
+
+
+
+mission "Nanachi 4"
+	name "Protect Nanachi"
+	description "Bring Nanachi's ship to Prime, keeping her safe from a pirate fleet that attacked a convoy heading to Hai space, then head back to <planet>."
+	source "Allhome"
+	stopover "Prime"
+	deadline 14
+	to offer
+		has "Nanachi 3: done"
+	on offer
+		conversation
+			`The bar is just as busy as the docks. Many captains here must have transported something for the festival. "Captain <last>, captain Nanachi!" shouts Narancia, beckoning the two of you to join her and the others at their table. You learn that they were inspired by the Hai's own March festival, and wanted to bring small parts of human culture to Hai space.`
+			`	"They can't go to our planets and experience it for themselves yet, so we thought about bringing it to them!" Vanessa says.`
+			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the European renaissance!" Xavier adds.`
+			`	Afterwards the conversation topic shifts to you as a captain, as you describe your journey after buying your first ship, and how you met Nanachi. You're about to order another drink for the group when a grizzled old man barges into the bar, and starts shouting: "The art convoy has been attacked! The freighter managed to escape but they're stranded on Prime now, and the pirates are still out there, patrolling the route to the wormhole!"`
+			`	Xavier stands up with a shocked expression. "How? We had a Leviathan escorting them!"`
+			`	After catching his breath, the man at the door responds. "The pirates must have been stealing Hai tech from ships going back home. The freighter's captain said they shredded the escort!" Chaos spreads amongst the other captains, and the group sitting with you is in anguish.`
+			choice
+				`	"Are those art pieces really that valuable?"`
+				`	"Can't you just postpone the festival until the pirates are taken care of?"`
+			`	Xavier collapses back to the table, holding his head in both hands. "Those paintings and sculptures were our main attraction! They're worth tens of millions! It'll take a month before the Navy ever notices the pirates there and makes it safe for the convoy to come here. The art exhibition is scheduled two weeks from now!"`
```

Comment:
>They're worth tens of millions!

Depending on the specific pieces involved and given the age mentioned, they may be worth billions, or even be priceless. The player themselves needs a net worth of 2 billion to even get to this point, so perhaps give more emphasis to the importance of the art. Maybe even don't mention that they were the main attraction, with Xavier being more concerned about the historical value of the art.

(At the very least, change "tens of millions" to "billions". The rest of the suggestion could be undesired depending on how we want Xavier and Vanessa to be characterized.)